### ISSUE TITLE
Peer Review - Kyungrok Park 

### ISSUE THREAD

**Issue by KyungrokP** [ZixuanYangFrank/Shelter#5]
- paper.qmd
* Abstract: I found that there is no result or suggestion from the research mentioned in abstract, I highly recommend to include the result or the final suggestion from your research 
* Introduction: It seems like there is no sentences for structure of the paper, it would be better to mention the structure so that readers can understand the rough context of your research 

-data
*raw_data.csv/Daily shelter overnight occupancy.csv: these csv files seem too large, we cannot see the dataset. it would be better to save as parquet file so that the dataset can be shown. 

-Readme.txt
I would recommend to leave an url or the website where the data(resources) comes from such that the readers can understand broader context of your research

In summary, the paper doesn't seem completed, but the data structure and the overall structure of the repo and file seem organized, the motivation is clearly stated and understandable, but the result and finding from the research seems very vague
